src/Components/Dialogs/GetPerson/GetElement/getElement.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import { I18Y } from '../../../../core/constants';
+
+import style from '../getPerson.module.css';
+
+const getElemen = (context, items) => context.map((elem) => <div><NavLink to={`${I18Y.path[0]}${elem}`} key={elem.toString()} className={style.item}>{items[elem].name}</NavLink></div>);
```

Comment:
"path" does not belong to I18Y which is a translation mapping object.
Moreover, storing "path" as an array is not convenient, object is way better for such purpose.

Reply:
I created new Object called Path.